### ISSUE TITLE
Allow `script_execute` and `script_execute_ext` to call constructors

### ISSUE THREAD

**Issue by rwkay** [YoYoGames/GameMaker-Bugs#7995]
### Is your feature request related to a problem?

With the addition of the error check to disallow constructors from being called normally - there is a requirement for library writers to be able to call a constructor with an array (in the way that `script_execute_ext` allows).

This will allow for more dynamic code to created without having a tower of doom to support different numbers of arguments.

### Describe the solution you'd like

Allow the following code to work 

```


function blah() constructor 
{
    baby = 5;
}

function foo() : blah() constructor 
{
    one = 10;
}

function bar() constructor
{
    two = 20;
    foo = new foo();
}

var a = {};
with( a ) {
    show_debug_message( $"{instanceof(self)} -> {a}" );
    script_execute( foo );
    show_debug_message( $"{instanceof(self)} -> {a}" );
    script_execute( bar );
    show_debug_message( $"{instanceof(self)} -> {a}" );
}

var b = {};
with( b ) {
    show_debug_message( $"{instanceof(self)} -> {b}" );
    script_execute_ext( foo );
    show_debug_message( $"{instanceof(self)} -> {b}" );
    script_execute_ext( bar );
    show_debug_message( $"{instanceof(self)} -> {b}" );
}
```

Project attached - 
[Bug-7995.zip](https://github.com/user-attachments/files/17360538/Bug-7995.zip)

### Describe alternatives you've considered

`constructor_call` and other new functions - but we do not want to add a new function at this time

**Comment by rwkay** [comment]
Added in 2024.11

**Comment by YYBartT** [comment]
Documented the new behaviour on the `script_execute()` and `script_execute_ext()` manual pages. Also added an additional code example on each page.

**Comment by scott-dunbar** [comment]
Verified fixed in  IDE v2024.1100.0.661  Runtime v2024.1100.0.687

Everything appears to be working as expected and documented in this ticket
